cypress/integration/RadioGroup.spec.ts

```diff
@@ -0,0 +1,40 @@
+import * as h from '../helpers';
+
+const getInidivualRadio = (index: number) => {
```

Comment:
There's a typo here but I think you can simplify this and let the params do the talking.

```suggestion
const getRadio = (index: number) => {
```